library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -18,6 +18,18 @@ namespace Extensions {
 namespace HttpFilters {
 namespace PlatformBridge {
 
+namespace {
+void replaceHeaders(Http::HeaderMap& headers, envoy_headers c_headers) {
+  headers.clear();
+  for (envoy_header_size_t i = 0; i < c_headers.length; i++) {
```

Comment:
Does `length` get recomputed on each iteration? If so, I wonder if we should set that to a variable above instead

Reply:
In this case `length` translates to an offset in a struct, so it's equivalent to a variable. In either case the value is probably already optimized into a register.